fpsim/locations/kenya.py

```diff
@@ -1212,10 +1165,10 @@ def education_distributions():
 
 # %% Make and validate parameters
 
-def make_pars():
-    '''
+def make_pars(use_empowerment=False, use_education=False, use_urban=False, use_partnership=False, seed=None):
```

Comment:
I've "unfolded" the `use_empowerment` flag into four (4) because empowerment, urban and partnership data are independent of each other. Education requires urban information at least for kenya which is the only country with all these new streams of data, but this won't necessarily be true for every other location doesn't have to. I don't think the current use of flags in make_pars() is ideal. It's kinda ugly really and to me indicates that what's done within a country.py file may need some refactoring.

If we added another stream of data in a country.py file, and we have 10 country files, we'd need to modify 10 more files to flag whether that stream is implemented or not. I feel that checking whether a particular data stream exists for a location needs to be handled somewhere else, at a higher level. Not sure where yet, perhaps in parameters.py, or the sim.py. 

make_pars() used to have no args because it was supposed to agnostic to user-defined parameters in `pars`, it simply ingested (loading, cleaning and some minor preprocessing like interpolation) all the data streams available to that country and return them to pars.